vitessce/wrappers.py

```diff
@@ -399,6 +399,7 @@ def __init__(self, adata=None, adata_url=None, expression_matrix=None, matrix_ge
         :param list[str] mappings_obsm_names: Overriding names like `['UMAP', 'PCA'] for displaying above scatterplots
         :param list[str] mappings_obsm_dims: Dimensions along which to get data for the scatterplot, like [[0, 1], [4, 5]] where [0, 1] is just the normal x and y but [4, 5] could be comparing the third and fourth principal components, for example.
         :param dict request_init: options to be passed along with every fetch request from the browser, like { "header": { "Authorization": "Bearer dsfjalsdfa1431" } }
+        param str gene_alias: A column name of gene names with which to display instead of the standard index in `var` of the AnnData store.
```

Comment:
```suggestion
        :param str gene_alias: The name of a column containing gene names, instead of the default index in `var` of the AnnData store.
```
Might be easier to read? But you know more about what's actually going one, so if I'm confused, please ignore!